tests/cephadm/test_cephadm_ansible_wrapper.py

```diff
@@ -32,27 +32,26 @@ def validate_configs(config):
     mon_node = module_args.get("mon_node")
     daemon_id = module_args.get("daemon_id")
     daemon_type = module_args.get("daemon_type")
-    osd_node = module_args.get("osd_node")
+    node = module_args.get("host")
     label = module_args.get("label")
+
     if module == "cephadm_bootstrap" and not mon_node:
         raise ConfigNotFoundError(
             "'cephadm_bootstrap' module requires 'mon_node' parameter"
         )
 
-    elif module in ["ceph_orch_host", "ceph_orch_apply"] and not osd_node and not label:
+    elif module == "ceph_orch_apply" and not node and not label:
         raise ConfigNotFoundError(
-            f"'{module}' module requires 'osd_node' and 'label' parameter"
+            f"'{module}' module requires 'host' and 'label' parameter"
         )
 
     elif module == "ceph_orch_daemon" and (not daemon_id or not daemon_type):
         raise ConfigNotFoundError(
             "'ceph_orch_daemon' module requires 'daemon_id' and 'daemon_type' parameter"
         )
 
-    elif module == "ceph_orch_host" and not osd_node:
-        raise ConfigNotFoundError(
-            "'ceph_orch_host' module requires 'osd_node' parameter"
-        )
+    elif module == "ceph_orch_host" and not node:
+        raise ConfigNotFoundError("'ceph_orch_host' module requires 'host' parameter")
```

Comment:
`host` or `node` ?

Reply:
The variable used in the code is node, whereas in the suite we are using 'host'. That's why have added host in this error log

Reply:
Noted. Changes made

Reply:
ok